src/CMakeLists.txt

```diff
@@ -80,9 +79,12 @@ if(BUILD_P11)
     endif(TEST_PKCS11_MODULE_PATH)
 endif(BUILD_P11)
 # set the name of the executable
-add_executable(aktualizr main.cc)
+if(BUILD_OSTREE)
+ add_executable(aktualizr main.cc ostree.cc)
+else(BUILD_OSTREE)
+ add_executable(aktualizr main.cc)
+endif(BUILD_OSTREE)
```

Comment:
This is super-trivial, but the indenting is inconsistent. Should be four spaces. The vim magic on the last line echoes that.